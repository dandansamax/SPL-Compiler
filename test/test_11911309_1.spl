int test_1_r01(int a, int b)
{
    int c=2;
  if (a > b)
  {
    return a;
  }
  else
  {
    return b;;
  }
}