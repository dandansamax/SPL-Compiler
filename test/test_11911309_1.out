Error type B at Line 3: Redundant type
